libs/dh/metering-point/feature-chargelink/src/components/create.ts

```diff
@@ -133,7 +120,7 @@ export class DhMeteringPointCreateChargeLink extends WattTypedModal {
     startDate: this.fb.control<Date | null>(null, Validators.required),
   });
 
-  selectedType = signal<ChargeType | null>(null);
+  selectedType = model<ChargeType | null>(null);
```

Comment:
nit: this does not need to be a model as far as I understand. it can still be a regular signal 🙂

Reply:
I´ll play around with it and see :-)